tests/common.py

```diff
@@ -61,8 +61,9 @@ def prepare_single_topology_initial_state(st: SingleTopology, host_config: Optio
     host = None
     if host_config is not None:
         host = rbfe.setup_optimized_host(st, host_config)
+    is_complex_leg = host_config is not None and len(host_config.conf) == host_config.num_water_atoms
```

Comment:
I'm probably missing something here, but this looks to me like it should be `is_solvent_leg`? (all host atoms are waters?)

Reply:
You aren't missing anything, that check is wrong. Fixed in https://github.com/proteneer/timemachine/pull/1346/commits/952dc8b069b28014c4b5d232ab8934880b21045f